ci/jobs/satellite6-pulp-upgrade.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
+    name: satellite6-pulp-upgrade
+    node: f23-vanilla-np
+    wrappers:
+        - ansicolor
+        - ssh-agent-credentials:
+            users:
+                - '044c0620-d67e-4172-9814-dc49e443e7b6'
+    builders:
+        - shell: |
+            set +e
```

Comment:
Should `set +e` be used on the entire block? What are the commands you are expecting to fail here? It will be better to just wrap into `set +e` and `set -e` those commands.